src/Chip.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
+import Icon from './Icon';
 
 const Chip = ({ children, close, className, ...props }) => {
   return (
     <div className={cx('chip', className)} {...props}>
       {children}
-      {close ? <i className="close material-icons">close</i> : null}
+      {close && <Icon className="close material-icons">close</Icon>}
```

Comment:
Hey @CloudPower97, just realised that `<Icon/>` component already includes these classes
```suggestion
      {close && <Icon>close</Icon>}
```
would suffice

Reply:
Whoops, you are right, It was my oversight